src/Library/Plugin/UpdateData.php

```diff
@@ -194,14 +194,24 @@ public function getResponseData() {
 	 * @return Response
 	 */
 	public function fetchResponseData( $force = false ) {
+		global $wp_filter;
 		$is_timely_updates  = apply_filters( 'boldgrid_backup_is_timely_updates', false );
 		$plugin_information = array();
 		$delay_time         = $force ? 0 : 3;
 		$delayFetchingData  = ( $this->getAgeOfTransient() < $delay_time );
 		if ( $is_timely_updates && ! $delayFetchingData ) {
+			/**
```

Comment:
https://make.wordpress.org/core/handbook/best-practices/inline-documentation-standards/php/#5-2-multi-line-comments

> ```
> /*
>  * This is a comment that is long enough to warrant being stretched over
>  * the span of multiple lines. You'll notice this follows basically
>  * the same format as the PHPDoc wrapping and comment block style.
>  */
> ```
> 
> **Important note**: Multi-line comments must not begin with /** (double asterisk) as the parser might mistake it for a DocBlock. Use /* (single asterisk) instead.

Reply:
OOOH. Shoot. I forgot about the double asterisk rule